contentcuration/contentcuration/static/js/edit_channel/queue/hbtemplates/queue_item.handlebars

```diff
@@ -1,22 +1,36 @@
 <div> <!-- Added to make jquery sortable more robust -->
-<input type="checkbox" id="queue_file_{{node.id}}" {{#if checked}}checked{{/if}}></input>
-<label for="queue_file_{{node.id}}" class="queue_item tog_folder">
-	<div class="queue-options pull-right">
-		<a class="btn content-options-dropdown" role="button" rel="popover">
-			<i class="material-icons">more_horiz</i>
-		</a>
-	</div>
-	<div class="queue_item_title title truncate {{#unless isfolder}}file_item{{/unless}}" title="{{node.title}}">
-		{{#unless isfolder}}<i class="content-item-icon material-icons">{{{get_icon node.kind}}}</i>&nbsp;{{/unless}}
+{{#unless is_segment}}
+	<input type="checkbox" id="queue_file_{{node.id}}" {{#if checked}}checked{{/if}}></input>
+{{/unless}}
+
+<label for="queue_file_{{node.id}}" class="queue_item tog_folder {{#if is_segment}}segment{{/if}}">
+	{{#unless is_segment}}
+		<div class="queue-options pull-right">
+			<a class="btn content-options-dropdown" role="button" rel="popover">
+				<i class="material-icons">more_horiz</i>
+			</a>
+		</div>
+  {{/unless}}
+
+	<div class="queue_item_title title truncate {{#if is_segment}}segment{{/if}}" title="{{node.title}}">
+		{{#if is_segment}}
+			{{#if node.thumbnail}}
+				<img class="content-item-thumbnail" aria-hidden="true" src="{{node.thumbnail}}" />
+			{{else}}
+				<span class="content-item-thumbnail">&nbsp;</span>
+			{{/if}}
+		{{/if}}
+		{{#unless is_segment}}<i class="content-item-icon material-icons">{{{get_icon node.kind}}}</i>&nbsp;{{/unless}}
 		{{node.title}}
```

Comment:
I wonder if it makes sense to add a link to the original channel here? Something like `<a href="/channels/{{channel.id}}/view" target="_blank">[link icon]</a>`?

Reply:
Great idea, done!